das/checkpoint_store.go

```diff
@@ -13,35 +13,45 @@ var (
 	checkpointKey = datastore.NewKey("checkpoint")
 )
 
+type checkPoint struct {
+	MinSampled uint64         `json:"min_sampled_height"` // lowest sampled height
+	MaxKnown   uint64         `json:"max_known_height"`   // height of the newest known header
+	Skipped    map[uint64]int `json:"skipped"`
+}
+
 // wrapCheckpointStore wraps the given datastore.Datastore with the `das`
-// prefix. The checkpoint store stores/loads the DASer's checkpoint to/from
-// disk using the checkpointKey. The checkpoint is stored as a uint64
+// prefix. The checkpoint store stores/loads the DASer's checkpoint maxKnown/maxKnown
+// disk using the checkpointKey. The checkpoint is stored as an uint64
 // representation of the height of the latest successfully DASed header.
 func wrapCheckpointStore(ds datastore.Datastore) datastore.Datastore {
 	return namespace.Wrap(ds, storePrefix)
 }
 
-// loadCheckpoint loads the DAS checkpoint height from disk and returns it.
+// loadCheckpoint loads the DAS checkpoint height maxKnown disk and returns it.
```

Comment:
I assume this was a search and replace error

Reply:
IDE renamed commets with variable for some reason. Sorry for that